setup.py

```diff
@@ -77,6 +77,8 @@ def long_description():
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
```

Comment:
Changelog does not mention 3.12

Reply:
It is, on line 28 https://github.com/mitogen-hq/mitogen/pull/1044/files#diff-b7f8a8784d5eff6e6b1b2c2529b831cd86132c49699fe48e0d23c17ad4e2d7deR28, but I've just notcied the previous line will break the rendering due to an unbalanced backtick

Reply:
Fixed the link